### PR TITLE
Surfacing method to get Completion Symbol

### LINKED ISSUE #10812: Surfacing method in the language service to get Completion Symbol
Need a method to surface the completion symbol instead of details for further extensibility of completion items. 

getCompletionEntrySymbol(fileName: string, position: number, entryName: string): Symbol;

### PR DESCRIPTION
**Pull request by microsoftsam**
Fixes #10812

Surfacing method to get Completion Symbol instead of details for better extensibility.

### CODE REVIEW

File: src/services/completions.ts [hunk 1]
```diff
@@ -736,6 +736,26 @@ namespace ts.Completions {
...
+        // Compute all the completion symbols again.
+        const completionData = getCompletionData(typeChecker, log, sourceFile, position);
+        if (completionData) {
+            const { symbols, location } = completionData;
+
+            // Find the symbol with the matching entry name.
+            // We don't need to perform character checks here because we're only comparing the
+            // name against 'entryName' (which is known to be good), not building a new
+            // completion entry.
+            const symbol = forEach(symbols, s => getCompletionEntryDisplayNameForSymbol(typeChecker, s, compilerOptions.target, /*performCharacterChecks*/ false, location) === entryName ? s : undefined);
```
Comment (mhegazy) [inline src/services/completions.ts]: just return the result of forEach. it will return a `symbol|undefined` which is what you want.

**Comment by mhegazy** [comment]
one small style comment, otherwise we are good to go.

**Comment by microsoftsam** [comment]
Okay, I'll get that in right away.

**Comment by mhegazy** [comment]
thanks!

**Pull request merged**